### ISSUE TITLE
Parsing error

### ISSUE THREAD

**Issue by emilyhawkins-drizly** [looker-open-source/look-at-me-sideways#61]
hi @fabio-looker - 

we're getting this parsing error 
```
Parsing error: Object.fromEntries is not a function
```

and i can't tell where it's coming from. 

im not seeing any errors on the CLI output itself..
```
Run lams --reporting=save-yes --source **/{views/core/catalog_items/dim_catalog_items.view.lkml,manifest.lkml} || echo "ERROR=true" >> $GITHUB_ENV
Parsing project...
> Issues occurred during parsing (containing files will not be considered. See messages for details.)
> Parsing done!
Checking rules... 
> E1
> E2
> F1
> F2
> F3
> F[4](https://github.com/DrizlyInc/data-looker/runs/5142238622?check_suite_focus=true#step:7:4)
> K1-2-3-4
> T1
> T2-[10](https://github.com/DrizlyInc/data-looker/runs/5142238622?check_suite_focus=true#step:7:10)
> Rules done!
Writing issues.md...
> issues.md done
```
<img width="818" alt="Screen Shot 2022-02-10 at 9 41 28 AM" src="https://user-images.githubusercontent.com/50426908/153430633-10546082-ac31-4fd3-b9d0-04e691d27a10.png">

any thoughts on what could be going wrong here?

**Comment by fabio-looker** [comment]
Hi @emilyhawkins-drizly ! This is probably closely related to https://github.com/looker-open-source/look-at-me-sideways/issues/58 , but seems a bit different

In the near term, pinning your LAMS to v1 should hopefully hold you over.

Would you also be able to send an erroring LookML file to me ( fabble@google.com ) so I can make a reduced test case from it?

**Comment by emilyhawkins-drizly** [comment]
Thanks @fabio-looker, will do!

**Comment by fabio-looker** [comment]
Thanks for bearing with me.

- Both the runtime error and reported parsing issues should be fixed in [v2.0.3](https://www.npmjs.com/package/@looker/look-at-me-sideways/v/2.0.3)
- **The default output format is different from v1, so even with these bugfixes v2 is not a drop-in replacement for v1**
- Docs have been updated with [an article about the contents of the v2 release](https://looker-open-source.github.io/look-at-me-sideways/release-notes/v2)
- To prevent similar issues in the future, I have updated any installation docs that install from NPM to include an explicit reference a major version in the installation command, and added tests to ensure these docs remain in-sync with the currently published major version.

If you still experience any unexpected errors, let me know and I will re-open the ticket.
